### DISCUSSION TITLE
A couple ideas

### DISCUSSION THREAD

**Discussion by Yohn** [CorentinTh/it-tools#257] [Ideas]
I saw you said to let ya know if I have any other ideas earlier, so here's a few that could possibly add onto what you already have. 

 * **Meta Tag Generator** A gen to give us the meta tags for open graph, twitter, facebook. I know theres a lot that could be added there as well.
 * **Headers** Could add onto that and add the headers for security policies and prevention in raw or meta tag form
 * **Pretty and Minify Code:** Just now I wanted a way to pretty print a minified CSS code and didn't see that there - did see the SQL and JSON though which is good, maybe combine them, and put other langues all there, - SASS, HTML and possibly into other languages if you find a way - and then maybe have them go both ways, to pretty print and minify.
 * **Color Scheme** This could be another can of worms, but to keep it simple having a way to put in a color code or a picker and then what colors go with that color picked. Something like [W3Schools](https://www.w3schools.com/colors/colors_schemes.asp) but allowing you to put in the color code instead of only being able to select from the wheel. I do feel like there was a few other color schemes that could be put in place to say, "this color could go well with these colors" but I don't remember what they all are.

**Comment by CorentinTh** [comment]
Thanks for those ideas 🙏🏻 
I'll try to implement some of them this next week 👌🏻 

Can you precise what you have in mind for the *headers* one?

**Reply by Yohn** [reply]
Something to help build the .htaccess or windows web.config files or add the meta tags to html for the headers being set to help prevent, or enable cors and xss.

Heres some references
https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS
https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP
https://ole.michelsen.dk/blog/secure-your-website-with-content-security-policy/

**Comment by CorentinTh** [comment]
The open graph metadata generator is live !
-> https://it-tools.tech/og-meta-generator

If needed, I've create a npm package to simply generate those metadata : oggen https://github.com/CorentinTh/oggen

![screely-1660924424491](https://user-images.githubusercontent.com/25065347/185659783-96a519a5-51a9-4cef-993d-f24b023c4e66.png)

**Comment by bxj0905** [comment]
Add a language toggle button. I want to finish the translation of Chinese, the buttons at the top will be easy to switch
